lib/mayaUsd/fileio/primWriterRegistry.h

```diff
@@ -109,7 +142,14 @@ struct UsdMayaPrimWriterRegistry
     ///
     /// If there is no writer plugin for \p mayaTypeName, returns nullptr.
     MAYAUSD_CORE_PUBLIC
-    static WriterFactoryFn Find(const std::string& mayaTypeName);
+    static WriterFactoryFn Find(
+        const std::string&          mayaTypeName,
+        const UsdMayaJobExportArgs& exportArgs,
+        const MObject&              exportObj);
+
+    /// \brief Check for external primWriter for \p mayaTypeName.
+    MAYAUSD_CORE_PUBLIC
+    static void Poke(const std::string& mayaTypeName);
 
```

Comment:
IDK if a better description and function name could be found. I read both the docs and function name and could not figure what the function did. The fact it returns nothing is adding to the confusion as it is unclear what the result of calling this is.  Since it seems to trigger scanning plugins for support for that Maya type, the description and function name could hint at that.